spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/web/ApplicationsController.java

```diff
@@ -49,15 +52,23 @@ public class ApplicationsController {
 
 	private final ApplicationRegistry registry;
 
-	public ApplicationsController(ApplicationRegistry registry) {
+	private final ApplicationEventPublisher publisher;
+
+	public ApplicationsController(ApplicationRegistry registry, ApplicationEventPublisher publisher) {
 		this.registry = registry;
+		this.publisher = publisher;
 	}
 
 	@GetMapping(path = "/applications", produces = MediaType.APPLICATION_JSON_VALUE)
 	public Flux<Application> applications() {
 		return registry.getApplications();
 	}
 
+	@PostMapping(path = "/applications", produces = MediaType.APPLICATION_JSON_VALUE)
+	public void refreshApplications() {
```

Comment:
I know this has already been merged, but I`m no sure if POST to /applications is intuitive for triggering discovery.

I would expect to send some data and add a new application on that method+path (like registering). That would also match the DELETE below where we unregister an application.